test/createStore.spec.js

```diff
@@ -45,6 +46,35 @@ describe('createStore', () => {
     }]);
   });
 
+  it('should warn if passed initial state has an unknown shape', () => {
+    const bootstrapData = {
+      pool: 1000,
```

Comment:
I think `counter` and `todos` would be more inline with the tests we have so far :-)